validator/node/node.go

```diff
@@ -130,10 +130,12 @@ func (s *ValidatorClient) registerClientService(ctx *cli.Context) error {
 	endpoint := ctx.GlobalString(types.BeaconRPCProviderFlag.Name)
 	keystoreDirectory := ctx.GlobalString(types.KeystorePathFlag.Name)
 	keystorePassword := ctx.String(types.PasswordFlag.Name)
+	logValidatorBalances := ctx.GlobalBool(types.DisablePenaltyRewardLogFlag.Name)
```

Comment:
```suggestion
	logValidatorBalances := !ctx.GlobalBool(types.DisablePenaltyRewardLogFlag.Name)
```

Reply:
How come you put the negation symbol the rather than in the if conditional. ? Globalbool already returns false if it's not found. Wouldn't make more sense to add the negation symbol at the conditional: `if !v.LogValidatorBalances {
 ` ? It would seem easier to read and would flow better in the long term.

Reply:
The name of the flag is `DisablePenaltyRewardLogFlag`, if this flag isnt used, then the assumption is that we want to log validator balances. Since the name of our var here is `logValidatorBalances` and implies the positive case, we negate it here.

Reply:
Ah ok, good point. I'll commit the suggestion now.